float64[3] A
float64 b
